src/selectedtests/app/evergreen.py

```diff
@@ -1,11 +1,13 @@
 """Web utilities for interacting with Evergreen API."""
+import inject
 
 from evergreen import EvergreenApi, Project
 from fastapi import HTTPException
 
 from selectedtests.evergreen_helper import get_evg_project
 
 
+@inject.autoparams()
```

Comment:
You'll see this in a few places; ideally it would be removed as well and we would maybe refactor this to be a bit more OOP oriented. With a refactor, you can just register injected constructors for each class, and they won't know about `inject` at all.